lib/cli/ui/prompt/options_handler.rb

```diff
@@ -0,0 +1,24 @@
+module CLI
+  module UI
+    module Prompt
+      # A class that handles the various options of an InteractivePrompt and their callbacks
+      class OptionsHandler
+        def initialize
+          @options = {}
+        end
+
+        def options
+          @options.keys
+        end
+
+        def add_option(option, &handler)
```

Comment:
This should probably also be just `option`. Adding a verb is generally not preferred for stylistic reasons in Ruby, just a conventions thing. I think we should add a `alias_method :option, :add_option` if you think `add_option` should stay.